.github/workflows/get-go-versions.yml

```diff
@@ -0,0 +1,93 @@
+name: Get Go versions
+on:
+  schedule:
+    - cron: '0 3,15 * * *'
+  workflow_dispatch:
+
+env:
+  TOOL_NAME: "Go"
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  find_new_versions:
+    name: Find new versions
+    runs-on: ubuntu-18.04
+    outputs:
+      versions_output: ${{ steps.Get_new_versions.outputs.version_number }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - id: Get_new_versions
+        name: Get new versions
+        run: ./helpers/get-new-tool-versions/get-new-tool-versions.ps1 -ToolName ${{ env.TOOL_NAME }}
+
+  check_new_versions:
+    name: Check new versions
+    runs-on: ubuntu-18.04
+    needs: find_new_versions
+    env:
+      TOOL_VERSIONS: ${{needs.find_new_versions.outputs.versions_output}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Check Versions
+        if: success() && env.TOOL_VERSIONS == ''
```

Comment:
Do we need the `success()` condition if we have `needs: find_new_versions`?

Reply:
Removed `success()` condition